docs/web_reference.rst

```diff
@@ -219,9 +219,9 @@ and :ref:`aiohttp-web-signals` handlers.
 
    .. attribute:: cookies
 
-      A multidict of all request's cookies.
+      A read-only dictionary-like object containing the request's cookies.
 
-      Read-only :class:`~multidict.MultiDictProxy` lazy property.
```

Comment:
I'd like to have an intersphinx reference to the actual type in the CPython docs. https://webknjaz.github.io/intersphinx-untangled/docs.python.org should help you construct it.

Reply:
:+1: Should be fixed by https://github.com/aio-libs/aiohttp/pull/5725/commits/767f6f516afde820101b2f4e6ad1625e3b12e017